src/index.js

```diff
@@ -140,6 +140,19 @@ var VERSION = process.env.npm_package_version;
  * @param {boolean} [commit] The user action to show on the PayPal review page. If true, a `Pay Now` button will be shown. If false, a `Continue` button will be shown.
  */
 
+/** @typedef {object} applePayCreateOptions The configuration options for Apple Pay.
+ *
+ * @param {string} [buttonStyle=`black`] Configures the Apple Pay button style. Valid values are `black`, `white`, `white-outline`. Default value is `black`.
```

Comment:
We shouldn't need to specify the default value, since that is indicated with `[buttonStyle=blank]`